### ISSUE TITLE
Node 20 types not supported

### ISSUE THREAD

**Issue by san1t1** [kettanaito/msw#1978]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `msw@2.1.2` for the project I'm working on.

I'm using 
- `"typescript": "^5.3.3"`
- `"@types/node": "^20.10.0"`
- `"msw": "^2.1.2"`

`tsc` tells me that `ResponseType` and `BodyInit` don't exist.
******

Here is the diff that solved my problem. I just made it vanish temporarily.

```diff
diff --git a/node_modules/msw/lib/core/RequestHandler-TRh8Eh4H.d.ts b/node_modules/msw/lib/core/RequestHandler-TRh8Eh4H.d.ts
index 365f8c1..9619e49 100644
--- a/node_modules/msw/lib/core/RequestHandler-TRh8Eh4H.d.ts
+++ b/node_modules/msw/lib/core/RequestHandler-TRh8Eh4H.d.ts
@@ -19,6 +19,7 @@ declare const getResponse: <Handler extends RequestHandler<RequestHandlerDefault
 }) => Promise<ResponseLookupResult | null>;
 
 interface HttpResponseInit extends ResponseInit {
+    // @ts-ignore
     type?: ResponseType;
 }
 declare const bodyType: unique symbol;
@@ -43,6 +44,7 @@ interface StrictResponse<BodyType extends DefaultBodyType> extends Response {
  * @see {@link https://mswjs.io/docs/api/http-response `HttpResponse` API reference}
  */
 declare class HttpResponse extends Response {
+    // @ts-ignore
     constructor(body?: BodyInit | null, init?: HttpResponseInit);
     /**
      * Create a `Response` with a `Content-Type: "text/plain"` body.
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by kettanaito** [comment]
Hi, @san1t1. This isn't an issue of MSW. In fact, this has been solved already. Take a look at https://github.com/mswjs/msw/issues/1836 and https://github.com/mswjs/msw/issues/1819.

TL;DR This was a bug in `@types/node`. It's since been fixed. Use the latest version of `@types/node`. If the issue remains, open it up in the DefinitelyTyped repo.
